src/Microsoft.AspNetCore.Razor.LanguageServer/CompletionItemExtensions.cs

```diff
@@ -37,5 +55,16 @@ public static ElementDescriptionInfo GetElementDescriptionInfo(this CompletionIt
 
             return ElementDescriptionInfo.Default;
         }
+
+        public static AttributeDescriptionInfo GetAttributeDescriptionInfo(this CompletionItem completion)
```

Comment:
It's weird to see the setter and getter have different names. `SetDescriptionData` and `GetAttributeDescriptionInfo`